resources/knative-build/charts/knative-build/templates/build.yaml

```diff
@@ -0,0 +1,443 @@
+apiVersion: policy/v1beta1
```

Comment:
Shouldn't that file be splitted up into multiple?

Reply:
done.